modules/01_3Scale_Architecture/Team1_3scale_OCP4_OCS_Install_Notes.adoc

```diff
@@ -0,0 +1,155 @@
+
+1. Install the OpenShift Container Storage operator to the `openshift-storage` namespace.
+2. Select update channel `stable-4.8`
+
+### 5. Create ClusterStorage resource using all 3 machines
+1. After installing the operator, you should see this message prompting you to create a `StorageCluster`:
+
+    Installed operator - operand required
+    
+2. Set `Requested Capacity` to `0.5 TiB`
+3. Select all three nodes
+4. All other settings can keep their defaults. Hit Create. If you see a `404` message, reload your browser.
+5. Change the default storage class
+
+### 6. Create APIManager resource
+1. Find the resource limits for this project:
+
+    # oc get limitranges
+    NAME                              CREATED AT
+    3scale-amp-core-resource-limits   2019-12-17T15:33:34Z
+    
+10. Edit the resource limit `3scale-amp-core-resource-limits` to change the resource limits as follows. The limit for memory as changed here to 32 GB to ensure the `redis-* pod`s of 3scale are deployed. This is only the max limit and does not affect the requested resources.
+    
+    # oc edit limitranges
+    
+    [...]
+    spec:
+      limits:
+      - default:
+          cpu: 500m
+          memory: 1536Mi
+        defaultRequest:
+          cpu: 50m
+          memory: 256Mi
+        max:
+          cpu: "3"
+          memory: 32Gi
+        type: Container
+      - max:
+          cpu: "2"
+          memory: 32Gi
+        type: Pod
+
+14. Create the `smtp.yml` configmap to configure SMTP access:
+
+    # vim smtp.yml
+    
+    kind: ConfigMap
+    apiVersion: v1
+    metadata:
+      name: smtp
+      labels:
+        app: 3scale-api-management
+        threescale_component: system
+        threescale_component_element: smtp
+    data:
+      address: 'smtp.gmail.com'
+      authentication: 'login'
+      domain: 'redhat.com'
+      openssl.verify.mode: 'false'
+      password: '<< your password>>'
+      port: '587'
+      username: '<< your userid>>'
+
+15. Add the configmap to `3scale-amp` namespace:
+
+    # oc create -f smtp.yml
+    
+16. Once Operator is installed and ready, get back to the terminal and add the APIManager:
+
+    # vim amp-s3.yml
+    
+    apiVersion: apps.3scale.net/v1alpha1
+    kind: APIManager
+    metadata:
+      name: apimanager
+    spec:
+      system:
+        fileStorage:
+          persistentVolumeClaim:
+            storageClassName: ocs-storagecluster-cephfs
+        redisResources:
+          limits:
+            memory: 6Gi
+      backend:
+        redisResources:
+          limits:
+            memory: 6Gi
+      wildcardDomain: <WILDCARD-DOMAIN>
+  
+    # oc create -f amp-s3.yml
+
+19. Wait for 10 mins for all the 3scale pods to be ready.
+20. Login to the Master URL and the `3scale-admin` tenant.
+21. Verify that you can open the Developer Portal and the Content is loaded correctly.
+
+## Next Steps
+`TODO:` In the next section, we will explore creating the tenants using operator.
```

Comment:
Please add tenant instructions here or remove this TODO

Reply:
Removed!